docs/test-specifications/token-service/TokenAirdropCancelTransaction.md

```diff
@@ -37,10 +37,11 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
 
 | Parameter Name          | Type                                             | Required/Optional | Description/Notes                                            |
 |-------------------------|--------------------------------------------------|-------------------|--------------------------------------------------------------|
-| senderAccountId        | string                                           | required          | The ID of the account that sent the airdrop.                 |
-| receiverAccountId      | string                                           | required          | The ID of the account that received the airdrop.             |
-| tokenId               | string                                           | required          | The ID of the token that was airdropped.                     |
-| serialNumbers           | list<string>                                            | optional          | The serial numbers of the NFTs to cancel airdrops.                                                            |
+| pendingAirdrops        | array                                            | required          | Array of pending airdrops to cancel                          |
+| pendingAirdrops[].senderAccountId        | string                                           | required          | The ID of the account that sent the airdrop                  |
+| pendingAirdrops[].receiverAccountId      | string                                           | required          | The ID of the account that received the airdrop              |
+| pendingAirdrops[].tokenId               | string                                           | required          | The ID of the token that was airdropped                      |
+| pendingAirdrops[].serialNumbers         | list<string>                                     | optional          | The serial numbers of the NFTs to cancel airdrops            |
```

Comment:
I think it might be better to describe the pending airdrop object in another markdown, since we might have to reuse it for token airdrop claim too